docs/about/stay-connected.rst

```diff
@@ -27,16 +28,25 @@ or moving them there if they develop into something bigger.
 
 Learn more about our :doc:`/slack`.
 
-Twitter
-~~~~~~~
+Job Board
+~~~~~~~~~
 
-We're pretty active on `Twitter <https://twitter.com/hashtag/writethedocs?f=tweets&vertical=default&lang=en>`__ as well, mostly on **#writethedocs**.
+Write the Docs now has a :doc:`/job-board`.
+This is the place to go if you want to hire or be hired by companies in the Write the Docs community.
+
+Promoted postings will also show up in our newsletter,
+so you can also keep track of jobs by subscribing there!
```

Comment:
What about something more along the lines of

We publicise promoted job posts in our newsletter, which goes out to XXX recipients.

You'll only see promoted jobs in the newseletter, so keeping track doesn't feel accurate.